WordPress/Classes/Utility/Sentry/WPCrashLogging.swift

```diff
@@ -0,0 +1,130 @@
+import Foundation
+import Sentry
+
+fileprivate let UserOptedOutKey = "crashlytics_opt_out"
+
+class WPCrashLogging {
+
+    fileprivate static let sharedInstance = WPCrashLogging()
+
+    static func start() {
+        // Create a Sentry client and start crash handler
+        do {
+            Client.shared = try Client(dsn: "https://7d2a517ff9de4433aa05d95b677943af@sentry.io/1438083")
```

Comment:
Is this a secret that shouldn't be in the public repo?

Reply:
The DSN isn't considered a secret. That said, it _is_ a configuration item. I was on the fence about this one, but since you mentioned it too, I'll move it over.

Reply:
This is now resolved.